src/app/_client-services/auth_client_service.ts

```
@@ -35,6 +35,18 @@ export class AuthClientService extends NextApiClientService {
 	}
 
+	static async remarkLogin({ address, wallet, remarkHash }: { address: string; wallet: EWallet; remarkHash: string }) {
+		return this.remarkLoginApi({
+			address,
+			wallet,
+			remarkHash
+		});
+	}
+
+	static async getRemarkLoginMessage({ address }: { address: string }) {
+		return this.getRemarkLoginMessageApi({ address });
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)